### ISSUE TITLE
Nonetype of param.grad.data

### ISSUE THREAD

**Issue by foury27** [KaiyangZhou/pytorch-center-loss#4]
Hi, I tried to use your package directly in my network. However, the param.grad is NoneType and param.grad.data cannot be got.. Could you help me to explain it?  Thanks!

**Comment by HanxunH** [comment]
Try`loss.backward()`
Before using`param.grad.data`
